Index: client/javascripts/controllers.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- client/javascripts/controllers.js	(date 1433760145000)
+++ client/javascripts/controllers.js	(revision )
@@ -11,11 +11,7 @@
 
     $scope.pageName = function () { return $location.path(); };
 
-    $scope.userSession = function () {
-        $scope.loggedIn = DOODService.gebruikerSessie.get({}, function () {
-            return $scope.loggedIn;
-        });
-    };
+    $scope.userSession = DOODService.gebruikerSessie.get();
 
 
 
